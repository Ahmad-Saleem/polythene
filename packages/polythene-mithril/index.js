export * from "polythene-mithril-base";
export * from "polythene-mithril-button";
export * from "polythene-mithril-fab";
export * from "polythene-mithril-icon";
export * from "polythene-mithril-raised-button";
export * from "polythene-mithril-ripple";
export * from "polythene-mithril-shadow";
export * from "polythene-mithril-svg";
