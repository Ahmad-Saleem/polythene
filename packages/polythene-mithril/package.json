{
  "name": "polythene-mithril",
  "version": "1.0.0-rc.1",
  "description": "",
  "main": "dist/polythene-mithril.js",
  "module": "dist/polythene-mithril.mjs",
  "scripts": {
    "clean": "rm -rf dist && mkdir dist",
    "rollup": "../../node_modules/rollup/bin/rollup -c ../../scripts/rollup.umd.js && ../../node_modules/rollup/bin/rollup -c ../../scripts/rollup.es.js",
    "build": "npm run rollup"
  },
  "files": [
    "dist",
    "README.md"
  ],
  "dependencies": {
    "polythene-mithril-base": "1.0.0-rc.1",
    "polythene-mithril-button": "1.0.0-rc.1",
    "polythene-mithril-card": "1.0.0-rc.1",
    "polythene-mithril-checkbox": "1.0.0-rc.1",
    "polythene-mithril-dialog": "1.0.0-rc.1",
    "polythene-mithril-dialog-pane": "1.0.0-rc.1",
    "polythene-mithril-fab": "1.0.0-rc.1",
    "polythene-mithril-icon": "1.0.0-rc.1",
    "polythene-mithril-icon-button": "1.0.0-rc.1",
    "polythene-mithril-list": "1.0.0-rc.1",
    "polythene-mithril-list-tile": "1.0.0-rc.1",
    "polythene-mithril-menu": "1.0.0-rc.1",
    "polythene-mithril-radio-button": "1.0.0-rc.1",
    "polythene-mithril-raised-button": "1.0.0-rc.1",
    "polythene-mithril-ripple": "1.0.0-rc.1",
    "polythene-mithril-shadow": "1.0.0-rc.1",
    "polythene-mithril-svg": "1.0.0-rc.1",
    "polythene-mithril-switch": "1.0.0-rc.1",
    "polythene-mithril-tabs": "1.0.0-rc.1",
    "polythene-mithril-textfield": "1.0.0-rc.1",
    "polythene-mithril-toolbar": "1.0.0-rc.1"
  },
  "author": "Arthur Clemens <arthurclemens@gmail.com> (http://visiblearea.com)",
  "license": "MIT"
}

