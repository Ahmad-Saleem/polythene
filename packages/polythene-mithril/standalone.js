export * from "./index";
import { addLayoutStyles, addFastClick } from "polythene-utilities";
import { addTypography, addRoboto } from "polythene-material-design";

addTypography();
addRoboto();
addLayoutStyles();
addFastClick();
