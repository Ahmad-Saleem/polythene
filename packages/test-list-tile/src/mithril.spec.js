import { runSnapshots } from "../../tests/scripts/mithril-snapshots";
import { renderer, Icon, ListTile } from "polythene-mithril";
import specTests from "./spec-tests.js";
import mithrilTests from "./tests-mithril.js";

runSnapshots({
  tests: specTests({ Icon, ListTile, renderer }).concat(mithrilTests),
  renderer
});
