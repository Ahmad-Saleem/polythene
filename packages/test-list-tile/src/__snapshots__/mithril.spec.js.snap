exports[`test Child node 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-icon pe-icon--regular\">
        <div class=\"pe-svg\">
          <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
          focusable=\"false\">
            <path d=
            \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
            </path>
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Dark tone class + light tone 1`] = `
"<div class=\"pe-list-tile pe-light-tone pe-list-tile--front\"
tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content pe-list-tile__content-front\">
      <div class=\"pe-icon pe-icon--medium\">
        <div class=\"pe-svg\">
          <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
          focusable=\"false\">
            <path d=
            \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
            </path>
          </svg>
        </div>
      </div>
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Dark tone class + light tone class 1`] = `
"<div class=\"pe-list-tile pe-list-tile--front pe-light-tone\"
tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content pe-list-tile__content-front\">
      <div class=\"pe-icon pe-icon--medium\">
        <div class=\"pe-svg\">
          <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
          focusable=\"false\">
            <path d=
            \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
            </path>
          </svg>
        </div>
      </div>
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test No options 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\"></div>
  </div>
</div>
"
`;

exports[`test Option: class 1`] = `
"<div class=\"pe-list-tile class-x\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\"></div>
  </div>
</div>
"
`;

exports[`test Option: className 1`] = `
"<div class=\"pe-list-tile className-x\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\"></div>
  </div>
</div>
"
`;

exports[`test Option: compact front (Icon) 1`] = `
"<div class=\"pe-list-tile pe-list-tile--front\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=
    \"pe-list-tile__content pe-list-tile__content-front pe-list-tile--compact-front\">
    <div class=\"pe-icon pe-icon--medium\">
        <div class=\"pe-svg\">
          <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
          focusable=\"false\">
            <path d=
            \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
            </path>
          </svg>
        </div>
      </div>
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: content 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-icon pe-icon--regular\">
      <div class=\"pe-svg\">
        <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\" focusable=
        \"false\">
          <path d=
          \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
          </path>
        </svg>
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: content 2`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div style=\"background: rgb(197, 225, 165); padding: 24px;\">
      The body lay naked and facedown, a deathly gray, spatters of
      blood staining the snow around it. It was minus fifteen
      degrees Celsius and a storm had passed just hours before.
    </div>
  </div>
</div>
"
`;

exports[`test Option: disabled url -- dark tone class 1`] = `
"<div class=\"pe-list-tile pe-list-tile--disabled\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: disabled url 1`] = `
"<div class=\"pe-list-tile pe-list-tile--disabled\">
  <a class=\"pe-list-tile__primary\" style=\"\" tabindex=\"0\" href=
  \"#!/\">
  <div class=\"pe-list-tile__content\">
    <div class=\"pe-list-tile__title\">
      Ancillary Justice
    </div>
  </div></a>
</div>
"
`;

exports[`test Option: element 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <a class=\"pe-list-tile__primary\" style=\"\">
  <div class=\"pe-list-tile__content\"></div></a>
</div>
"
`;

exports[`test Option: events 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Click me
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: front (Icon) 1`] = `
"<div class=\"pe-list-tile pe-list-tile--front\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content pe-list-tile__content-front\">
      <div class=\"pe-icon pe-icon--medium\">
        <div class=\"pe-svg\">
          <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
          focusable=\"false\">
            <path d=
            \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
            </path>
          </svg>
        </div>
      </div>
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: front (avatar) 1`] = `
"<div class=\"pe-list-tile pe-list-tile--front\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content pe-list-tile__content-front\">
      <div class=\"pe-icon pe-icon--large pe-icon--avatar\">
        <img src=
        \"http://arthurclemens.github.io/assets/polythene/examples/avatar-1.png\">
      </div>
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: highSubtitle 1`] = `
"<div class=\"pe-list-tile pe-list-tile--high-subtitle\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
      <div class=
      \"pe-list-tile__subtitle pe-list-tile__high-subtitle\">
        The body lay naked and facedown, a deathly gray, spatters
        of blood staining the snow around it. It was minus fifteen
        degrees Celsius and a storm had passed just hours before.
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: highSubtitle and compact 1`] = `
"<div class=
\"pe-list-tile pe-list-tile--compact pe-list-tile--high-subtitle\"
tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
      <div class=
      \"pe-list-tile__subtitle pe-list-tile__high-subtitle\">
        The body lay naked and facedown, a deathly gray, spatters
        of blood staining the snow around it. It was minus fifteen
        degrees Celsius and a storm had passed just hours before.
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: highSubtitle and front -- dark tone class 1`] = `
"<div class=\"pe-list-tile pe-list-tile--high-subtitle\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content pe-list-tile__content-front\">
      <div class=\"pe-icon pe-icon--large pe-icon--avatar\">
        <img src=
        \"http://arthurclemens.github.io/assets/polythene/examples/avatar-1.png\">
      </div>
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
      <div class=
      \"pe-list-tile__subtitle pe-list-tile__high-subtitle\">
        The body lay naked and facedown, a deathly gray, spatters
        of blood staining the snow around it. It was minus fifteen
        degrees Celsius and a storm had passed just hours before.
      </div>
    </div>
  </div><a href=\"#!/\" class=\"pe-list-tile__secondary\" tabindex=\"0\">
  <div class=\"pe-list-tile__content\">
    <div class=\"pe-icon pe-icon--regular\">
      <div class=\"pe-svg\">
        <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\" focusable=
        \"false\">
          <path d=
          \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
          </path>
        </svg>
      </div>
    </div>
  </div></a>
</div>
"
`;

exports[`test Option: highSubtitle and front 1`] = `
"<div class=\"pe-list-tile pe-list-tile--high-subtitle\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content pe-list-tile__content-front\">
      <div class=\"pe-icon pe-icon--large pe-icon--avatar\">
        <img src=
        \"http://arthurclemens.github.io/assets/polythene/examples/avatar-1.png\">
      </div>
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
      <div class=
      \"pe-list-tile__subtitle pe-list-tile__high-subtitle\">
        The body lay naked and facedown, a deathly gray, spatters
        of blood staining the snow around it. It was minus fifteen
        degrees Celsius and a storm had passed just hours before.
      </div>
    </div>
  </div><a href=\"#!/\" class=\"pe-list-tile__secondary\" tabindex=\"0\">
  <div class=\"pe-list-tile__content\">
    <div class=\"pe-icon pe-icon--regular\">
      <div class=\"pe-svg\">
        <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\" focusable=
        \"false\">
          <path d=
          \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
          </path>
        </svg>
      </div>
    </div>
  </div></a>
</div>
"
`;

exports[`test Option: highlight (themed color) 1`] = `
"<div class=
\"pe-list-tile pe-list-tile--highlight tests-list-tile-themed-highlight-list-tile\"
tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: hoverable 1`] = `
"<div class=\"pe-list-tile pe-list-tile--hoverable\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: id 1`] = `
"<div id=\"id-x\" class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\"></div>
  </div>
</div>
"
`;

exports[`test Option: indent 1`] = `
"<div class=\"pe-list-tile pe-list-tile--front\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content pe-list-tile__content-front\">
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: ink 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-ripple\"></div>
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: ink with ripple options 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-ripple\"></div>
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: secondary (Icon) (RTL) 1`] = `
"<div dir=\"rtl\">
  <div class=\"pe-list-tile pe-list-tile--subtitle\" tabindex=\"0\">
    <div class=\"pe-list-tile__primary\" style=\"\">
      <div class=\"pe-list-tile__content\">
        <div class=\"pe-list-tile__title\">
          Ancillary Justice
        </div>
        <div class=\"pe-list-tile__subtitle\">
          The body lay naked and facedown, a deathly gray, spatters
          of blood staining the snow around it. It was minus
          fifteen degrees Celsius and a storm had passed just hours
          before.
        </div>
      </div>
    </div>
    <div class=\"pe-list-tile__secondary\">
      <div class=\"pe-list-tile__content\">
        <div class=\"pe-icon pe-icon--medium\">
          <div class=\"pe-svg\">
            <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
            focusable=\"false\">
              <path d=
              \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
              </path>
            </svg>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: secondary (Icon) 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
  <div class=\"pe-list-tile__secondary\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-icon pe-icon--medium\">
        <div class=\"pe-svg\">
          <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
          focusable=\"false\">
            <path d=
            \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
            </path>
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: secondary (content) 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
  <div class=\"pe-list-tile__secondary\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-icon pe-icon--regular\">
        <div class=\"pe-svg\">
          <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
          focusable=\"false\">
            <path d=
            \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
            </path>
          </svg>
        </div>
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: secondary (url) 1`] = `
"<div class=\"pe-list-tile\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div><a href=\"#!/\" class=\"pe-list-tile__secondary\" tabindex=\"0\">
  <div class=\"pe-list-tile__content\">
    <div class=\"pe-icon pe-icon--medium\">
      <div class=\"pe-svg\">
        <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\" focusable=
        \"false\">
          <path d=
          \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
          </path>
        </svg>
      </div>
    </div>
  </div></a>
</div>
"
`;

exports[`test Option: selectable 1`] = `
"<div class=\"pe-list-tile pe-list-tile--selectable\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: selected 1`] = `
"<div class=\"pe-list-tile pe-list-tile--selected\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: style (colors) 1`] = `
"<div style=
\"color: rgb(255, 255, 255); background-color: rgb(239, 108, 0);\"
class=\"pe-list-tile pe-list-tile--front\" tabindex=\"0\">
  <div style=\"\" class=\"pe-list-tile__primary\">
    <div class=\"pe-list-tile__content pe-list-tile__content-front\">
      <div class=\"pe-icon pe-icon--medium\">
        <div class=\"pe-svg\">
          <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
          focusable=\"false\">
            <path d=
            \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
            </path>
          </svg>
        </div>
      </div>
    </div>
    <div class=\"pe-list-tile__content\" style=
    \"color: rgb(255, 255, 255); background-color: rgb(239, 108, 0);\">
    <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: subContent 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
      <div>
        <div style=
        \"background: rgb(197, 225, 165); margin-top: 12px;\">
          The body lay naked and facedown, a deathly gray, spatters
          of blood staining the snow around it. It was minus
          fifteen degrees Celsius and a storm had passed just hours
          before.
        </div>
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: subtitle 1`] = `
"<div class=\"pe-list-tile pe-list-tile--subtitle\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
      <div class=\"pe-list-tile__subtitle\">
        The body lay naked and facedown, a deathly gray, spatters
        of blood staining the snow around it. It was minus fifteen
        degrees Celsius and a storm had passed just hours before.
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: title 1`] = `
"<div class=\"pe-list-tile\" tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        ListTile with a very very very very very very very very
        very long title
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Option: url 1`] = `
"<div class=\"pe-list-tile\">
  <a class=\"pe-list-tile__primary\" style=\"\" tabindex=\"0\" href=
  \"#!/\">
  <div class=\"pe-list-tile__content\">
    <div class=\"pe-list-tile__title\">
      Ancillary Justice
    </div>
  </div></a>
</div>
"
`;

exports[`test Themed (color and font size) -- dark tone class 1`] = `
"<div class=
\"pe-list-tile pe-list-tile--front tests-list-tile-themed-list-tile\"
tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content pe-list-tile__content-front\">
      <div class=\"pe-icon pe-icon--medium\">
        <div class=\"pe-svg\">
          <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
          focusable=\"false\">
            <path d=
            \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
            </path>
          </svg>
        </div>
      </div>
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;

exports[`test Themed (color and font size) 1`] = `
"<div class=
\"pe-list-tile pe-list-tile--front tests-list-tile-themed-list-tile\"
tabindex=\"0\">
  <div class=\"pe-list-tile__primary\" style=\"\">
    <div class=\"pe-list-tile__content pe-list-tile__content-front\">
      <div class=\"pe-icon pe-icon--medium\">
        <div class=\"pe-svg\">
          <svg width=\"24\" height=\"24\" viewbox=\"0 0 24 24\"
          focusable=\"false\">
            <path d=
            \"M11.99 2C6.47 2 2 6.48 2 12s4.47 10 9.99 10C17.52 22 22 17.52 22 12S17.52 2 11.99 2zm4.24 16L12 15.45 7.77 18l1.12-4.81-3.73-3.23 4.92-.42L12 5l1.92 4.53 4.92.42-3.73 3.23L16.23 18z\">
            </path>
          </svg>
        </div>
      </div>
    </div>
    <div class=\"pe-list-tile__content\">
      <div class=\"pe-list-tile__title\">
        Ancillary Justice
      </div>
    </div>
  </div>
</div>
"
`;
